tests/ui/const-generics/generic_const_exprs/issue-105709.rs

```diff
@@ -0,0 +1,9 @@
+// build-pass
```

Comment:
This could use a better issue name. Perhaps something like `inline-const-in-const-generic-default` or something?